sqlglot/dialects/spark2.py

```diff
@@ -293,5 +283,10 @@ def columndef_sql(self, expression: exp.ColumnDef, sep: str = " ") -> str:
                 ),
             )
 
+        def struct_sql(self, expression: exp.Struct) -> str:
+            from sqlglot.generator import Generator
+
+            return Generator.struct_sql(self, expression)
```

Comment:
Can also add this in `TRANSFORMS` if we wanna avoid defining this method? It should take precedence.